moduleroot/CONTRIBUTING.md

```diff
@@ -156,24 +157,32 @@ If you already have those gems installed, make sure they are up-to-date:
 % bundle update
 ```
 
+## Running Tests
+
 With all dependencies in place and up-to-date we can now run the tests:
```

Comment:
comma after up-to-date, and I would change "we can now" to just "run the tests."